13_packages/main.go

```diff
@@ -57,6 +57,10 @@ import (
 	"fmt"
 
 	"github.com/riraum/forward/internal/geometry"
+
+	"github.com/riraum/forward/internal/person"
+
+	"github.com/MakeNowJust/heredoc/v2"
```

Comment:
now that we manage imports as well, have a look at https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Maybe can you change https://github.com/riraum/forward/blob/master/.github/workflows/ci.yaml to run `goimports` instead/alongside `go fmt` ?

Reply:
Otherwise I can do it, let me know :)

Reply:
Attempt done.

Reply:
Left some comments, let's continue the discussion there.